content/post/opera-ios.md

```diff
@@ -0,0 +1,38 @@
+<p style="text-align: center;">
+  <img src="/img/opera-ipfs-header.png" alt="IPFS in Opera Touch for iOS">
+</p>
+
+
+In 2020 we announced a big moment for IPFS: The first official support of IPFS protocol addressing in a major browser, when [Opera released IPFS support in their Android browser](https://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/). This was an important step in IPFS browser support generally, by building interest and momentum. We didn’t stop there, and we’ve got not just one… BUT TWO releases to share with you today!
+
+First, Opera has now added support for IPFS addressing to Opera Touch, their iOS browser.
+
+<p style="text-align: center;">
+  <img src="/img/opera-ios-wikipedia-short.png" alt="A screenshot of Wikipedia on IPFS in Opera Touch">
+</p>
+
+Second, support for IPFS addressing in Opera desktop browser for Windows, macOS, and Linux will be coming in their next release, currently planned for March 2021.
+
+With these releases, Opera will now support `ipfs://` and `ipns://` addressing across their browser product line on all major operating systems: Windows, macOS, Linux, iOS, and Android.
+
+## How do I use it?
+
+First, install Opera Touch on your iOS device. If you’re reading this on an iOS device, [click to install now](https://apps.apple.com/us/app/opera-touch-web-browser/id1411869974).
+
+<p style="text-align: center;">
+  <img src="/img/opera-ios-app-store-short.png" alt="Opera Touch in the iOS app store">
+</p>
+
+Then you can open content using IPFS protocol addresses, like the <a href="ipns://blog.ipfs.io">blog you’re reading now</a>, <a href="ipns://en.wikipedia-on-ipfs.org">Wikipedia</a>, or this <a href="ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi">Persian room guardian</a> by [Anya Boz](https://www.anyabozartist.com/the-persian-cat). You can easily upload files to IPFS using the [IPFS desktop application](https://docs.ipfs.io/install/ipfs-desktop/) or [Share.ipfs.io](https://share.ipfs.io).
+
+## How does it work?
+
+Opera Touch supports navigating to addresses for `ipfs://` and `ipns://` protocol schemes, which are handled by a remote HTTP gateway. By default the gateway used is `dweb.link`, which is operated by Protocol Labs. TODO: Can select/change gateway?
```

Comment:
TODO: Resolve this TODO line ☺️

Reply:
Not in 2.5.5, should be released with 2.5.6 hopefully this week (release testing ongoing).

Reply:
Fixed!